zenml/pipelines/training_pipeline.py

```diff
@@ -204,23 +203,26 @@ def get_tfx_component_list(self, config: Dict[Text, Any]) -> List:
         # EVALUATOR #
         #############
         if keys.TrainingSteps.EVALUATOR in steps:
-            from zenml.utils import source_utils
-            eval_module = '.'.join(
-                constants.EVALUATOR_MODULE_FN.split('.')[:-1])
-            eval_module_file = constants.EVALUATOR_MODULE_FN.split('.')[-1]
-            abs_path = source_utils.get_absolute_path_from_module_source(
-                eval_module)
-            custom_extractor_path = os.path.join(abs_path,
-                                                 eval_module_file) + '.py'
-            eval_step: TFMAEvaluator = TFMAEvaluator.from_config(
-                steps[keys.TrainingSteps.EVALUATOR])
-            eval_config = eval_step.build_eval_config()
-            evaluator = Evaluator(
-                examples=transform.outputs.transformed_examples,
-                model=trainer.outputs.model,
-                eval_config=eval_config,
-                module_file=custom_extractor_path,
-            ).with_id(GDPComponent.Evaluator.name)
+            evaluator_config = steps[keys.TrainingSteps.EVALUATOR]
+            # TODO: This check should ideally not happen here, integrate it
```

Comment:
Can you add [LOW], [MEDIUM], [HIGH] here?

Reply:
Added the MEDIUM tag